sass/partials/_time_to_retirement.scss

```diff
@@ -449,5 +519,4 @@ input[type="text"].form-control {
     position: absolute;
     left: 15%;
     right: 15%;
-}
-
+}
```

Comment:
Yeah you're probably right, and also we should break them into multiple partials, since there are pretty distinct sections.